realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +406,192 @@ public boolean isValid() {
         return this;
     }
 
+    // In
```

Comment:
There are `// Equal` and `// Not Equal` in this file. Should we delete them too?